default.xml

```diff
@@ -1,19 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
-  <remote name="github" fetch="https://github.com/" />
-  <remote name="gitlab" fetch="ssh://git@gitlab.bisdn.de/" />
-  <remote name="oe" fetch="git://git.openembedded.org/" />
-  <remote name="yocto" fetch="git://git.yoctoproject.org/" />
-  <!-- yocto layer -->
-  <project name="meta-cloud-services" path="poky/meta-cloud-services" remote="yocto" revision="dunfell"/>
-  <project name="meta-virtualization" path="poky/meta-virtualization" remote="yocto" revision="dunfell"/>
-  <project name="poky" path="poky" remote="yocto" revision="dunfell"/>
-  <!-- open embedded layer -->
-  <project name="meta-openembedded" path="poky/meta-openembedded" remote="oe" revision="dunfell"/>
-  <!-- open source bisdn layers -->
-  <project name="bisdn/bisdn-linux.git" path="poky/build-bisdn-linux" remote="github" revision="master"/>
-  <project name="bisdn/bisdn-onie-additions.git" path="poky/bisdn-onie-additions" remote="github" revision="master"/>
-  <project name="bisdn/meta-ofdpa.git" path="poky/meta-ofdpa" remote="github" revision="master"/>
-  <project name="bisdn/meta-open-network-linux.git" path="poky/meta-open-network-linux" remote="github" revision="master"/>
-  <project name="bisdn/meta-switch.git" path="poky/meta-switch" remote="github" revision="master"/>
+  <remote fetch="https://github.com/" name="github"/>
```

Comment:
The commit changes the order name/fetch to fetch/name and also removes a space. Similar changes have been made on other lines. Formatting changes should be in a separate commit along with some explanation.

Reply:
Because the base file is a (semi) handwritten file, while the one replacing it is an autogenerated one (repo discards all comments). There is no intermediary version with just the changes, or just the formatting.

Reply:
It would be easily possible to create an intermediate version to make things easier for reviewers. Even if no such service is offered, an explanation in the commit message and the PR is a must.

Reply:
I disagree, an explanation in the release process yes, but I don't this needs to be explained every time we do a release.

Reply:
Can we continue without this, and I will add it to the documentation process for future releases?

Reply:
I will just assume that I don't understand sufficiently what is going on and that you guys will sort it out.

Reply:
I added a commit message pointing to the generated .xml file.

Reply:
This isn't a stable link, the contents will change with every image build. This will be confusing when the commit hashes e.g. in a week do not match any more at all.

Reply:
COmmit message removed